src/components/AudioInfo.vue

```diff
@@ -80,6 +84,41 @@ export default defineComponent({
       });
     };
 
+    const setAudioInfoByScroll = (
+      query: AudioQuery,
+      delta_y: number,
+      type: string
```

Comment:
It may be clearer to define the type.
`    type InfoType = "speed" | "pitch" | "into";` and
```suggestion
      type: InfoType
```